### ISSUE TITLE
IJ

### ISSUE THREAD

**Issue by mulliganaceous** [JetBrains/JetBrainsMono#578]
As of now, the font is still missing the IJ glyph in Latin Extended-A.

The best design for the capital IJ should look like a broken U.

![JBĲĳ](https://user-images.githubusercontent.com/29824582/201829999-d6b5f994-1911-4f50-855b-54350b716bc0.png)

**Comment by philippnurullin** [comment]
Hi @mulliganaceous 
Thanks for the feedback. You are absolutely right, there is no such symbol in JetBrains Mono, and it is by design. This traditional ligature breaks the golden rule of monospaced typeface: never modify the number of symbols in the line. It merges two symbols into one and also merges the space taken by the symbols. 

Every ligature we have in JetBrains Mono is taking the same amount of space regardless of legation.

Why do you need this particular glyph?

**Comment by smups** [comment]
tl;dr: ĳ/Ĳ is very much the same as æ/Æ and œ/Œ: technically they're just two letters smushed together, but they are often considered to be a single letter as they don't make the sound as their constituent letters (at all) and must always be capitalised together.

Longer story: the combination ĳ has a special place in dutch orthography and spelling. It would be nice to have the option to use the characters ĳ/Ĳ for those who prefer to indicate that it is a single letter; like how the Danes often prefer to write æ over ae. 

In addition, in handwriting ĳ is always written as a single letter (like y/Y with dots on top). Some users might want to emulate that look by using the ĳ/Ĳ characters. For reference, this is what Dutch handwriting (as taught in school) looks like:

![wikipedia](https://upload.wikimedia.org/wikipedia/commons/3/3f/Schoonschrift.jpg)

The main difference between Danish æ and Dutch ĳ is that the status of ĳ as a single letter is much more contentious than the status of æ as a single letter. The situation is comparable to French œ, in the sense that computers don't have a separate key for œ/ĳ, and it's not incorrect or something to write the two as separate letters.

Right so this is why one might argue that ĳ is a single letter:

- It's pronounced as a single sound (except in loan words)
- It's always capitalised together. For example, the Amsterdam neighbourhood of Ĳburg (never Ijburg)
- In handwriting, it is always written as a single letter. Basically "y" with dots on top, for both majuscule and minuscule versions.
- It was used interchangeably with the letter y for much of dutch literary history

For different reasons, it's often thought of as two letters too:

- Historically, the ĳ is a doubling of the letter i. As in bliven vs bliift just like modern kopen and koopt. In the past, people often didn't bother with the dots on letters, so ii looked too similar to u. That's why they added a lil swirl on the second i to create ij.
- A dude in the 1920's said so
- Computers don't have a separate ij key (anymore)
- IJ as two separate letters is much more common in typeset documents, although dutch texts often make use of fonts with ligatures for ij/IJ

**Comment by smups** [comment]
Oh and btw, I think OP was asking for characters for the Ĳ (U+0132) en ĳ (U+0133), s.t. Dutch users can have access to nice letters without annoying non-dutch users with an unexpected/unwanted ligature.

**Comment by philippnurullin** [comment]
Hi @smups 
Thanks for the deep dive in the issue. 

How do Dutch users usually access the `Ĳ` `ĳ`?

**Comment by smups** [comment]
Most Dutch (Netherlands) users use a standard US QWERTY keyboard with dead keys or a compose key btw, AZERTY (the French layout) is also common in Belgium cuz of the whole bilingual thing they got going on.

**Comment by philippnurullin** [comment]
Sounds like a pain. )

If there is no ligation expected from `I+J` and `i+j` then the symbols can be added.

Thanks again for the very helpful and interesting conversation. The numbers on the Dutch handwriting manual are my absolute favorite. And "_A dude in the 1920's said so_" historically speaking, is a very solid argument. I mean, it really is. ))

**Comment by smups** [comment]
> If there is no ligation expected from I+J and i+j then the symbols can be added.

That'd be cool! I know it´s a bit of a niche issue (even amongst Dutch users), but it would make some of us (me) very happy if those symbols would be available :)

In case you're wondering about who exactly the dude from the 20's is that said so, I'm actually wrong about that lol, he (Kollewĳn) supported the usage of `y` over `ij`. Ĳ was already official at that point because another two dudes had said so 40 years prior (De Vries en Te Winkel, 1880's). South Africa actually implemented Kollewĳns spelling reforms, which is why contemporary Afrikaans has y instead of ĳ, and -lik instead of -lĳk (amongst other differences).

**Comment by smups** [comment]
Hi again!

Thank you so much for adding ĳ and Ĳ, they look great!

I tried out the dev. version of the font. I think it works very well. I really like what you did with the capitol Ĳ, I think it looks great, although I think it might not work so well at a distance/with a small font size.

This first example I think really shows that the ij ligature significantly improves readability - the word "dijing" especially looks a lot better with the new ĳ.

![image](https://user-images.githubusercontent.com/56522208/227244952-84abd796-4081-4036-ba86-5628655aeb04.png)

The next example I have is a comparison between an ad of the Rĳksmuseum with their own custom font and a recreation with the new ligatures:
![image](https://user-images.githubusercontent.com/56522208/227246098-438a6c4e-9a5a-4643-a6e3-9a007f53c230.png)
![image](https://user-images.githubusercontent.com/56522208/227246172-9d6a5ca4-3a48-4a4f-ab4e-102a65a3153f.png)

**Comment by mulliganaceous** [comment]
Saw this, quite surprised that "ij" is still up to consideration. Personally, jetbrains mono is also a great ux font!
